src/components/PoolingWidget/index.tsx

```diff
@@ -296,8 +296,9 @@ const PoolingInterface: React.FC = () => {
       txHash,
       txReceipt,
       txError,
+      nextStep,
     }),
-    [handleTokenSelect, selectedTokensMap, setSpread, spread, tokens, txError, txHash, txReceipt],
+    [handleTokenSelect, nextStep, selectedTokensMap, setSpread, spread, tokens, txError, txHash, txReceipt],
```

Comment:
While you are at it can you please change `prevStep, nextStep` to use `setStep(step => step +- 1)` and remove `[setStep, ~step~]` from deps